web-console/src/lib/components/MonacoEditorRunes.svelte

```diff
@@ -0,0 +1,133 @@
+<script lang="ts" module>
+  loader.config({ monaco: monacoImport, 'vs/nls': { availableLanguages: { '*': 'en' } } })
+
+  export const exportedThemes = Object.fromEntries(
+    Object.entries(import.meta.glob('/node_modules/monaco-themes/themes/*.json')).map(([k, v]) => [
+      k.toLowerCase().split('/').reverse()[0].slice(0, -'.json'.length).replaceAll(' ', '-'),
+      v
+    ])
+  )
+
+  export const nativeThemes = ['vs', 'vs-dark', 'hc-black']
+
+  export const themeNames: string[] = [...Object.keys(exportedThemes), ...nativeThemes].sort(
+    (a, b) => a.localeCompare(b)
+  )
+
+  import editorWorker from 'monaco-editor/esm/vs/editor/editor.worker?worker'
+  import jsonWorker from 'monaco-editor/esm/vs/language/json/json.worker?worker'
+  import cssWorker from 'monaco-editor/esm/vs/language/css/css.worker?worker'
+  import htmlWorker from 'monaco-editor/esm/vs/language/html/html.worker?worker'
+  import tsWorker from 'monaco-editor/esm/vs/language/typescript/ts.worker?worker'
+
+  self.MonacoEnvironment = {
+    getWorker(_: any, label: string) {
```

Comment:
do we have these filetypes/labels?

Reply:
This is basically first-party recommended copy-pasta to get all monaco-editor components to work nicely, these do not relate to used code editor languages (In our case SQL, Rust etc.)